web/geosearch/datapunt_geosearch/datasource.py

```diff
@@ -161,6 +162,11 @@ def get_variable_sql(self, table: str) -> Dict[str, sql.Identifier]:
             ),
         )
 
+    def get_db_crs(self) -> sql.Literal:
+        if self.crs is not None:
+            return sql.SQL(self.crs.split(":")[-1])
+        return sql.SQL("28992")
```

Comment:
Waarom wordt dit in een `sql.SQL` gezet, en is de return value een `sql.Literal`?

Reply:
goeie catch